ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -436,3 +436,18 @@ class BaremetalRedfishIPxeWholediskHttpLink(
     @utils.services('network')
     def test_ip_access_to_server(self):
         self.boot_and_verify_node()
+
```

Comment:
Should have 2 empty lines between 2 classes (run tox before pushing)

Reply:
ohk

Reply:
Done